sql-plugin/src/main/scala/org/apache/spark/sql/rapids/mathExpressions.scala

```diff
@@ -47,9 +47,14 @@ case class GpuAcos(child: Expression) extends CudfUnaryMathExpression("ACOS") {
 case class GpuToDegrees(child: Expression) extends GpuUnaryMathExpression("DEGREES") {
 
   override def doColumnar(input: GpuColumnVector): ColumnVector = {
-    withResource(Scalar.fromDouble(180d / Math.PI)) { multiplier =>
+    val tmp = withResource(Scalar.fromDouble(180d)) { multiplier =>
```

Comment:
nit: Can we add a note so no one tries to optimize this code and make it simpler again?